client/my-sites/pages/virtual-page/index.tsx

```diff
@@ -19,9 +20,68 @@ import { canCurrentUser } from 'calypso/state/selectors/can-current-user';
 import { setLayoutFocus } from 'calypso/state/ui/layout-focus/actions';
 import { setPreviewUrl } from 'calypso/state/ui/preview/actions';
 import Placeholder from '../placeholder';
-import type { SiteDetails } from '@automattic/data-stores';
+import type { SiteDetails, Template } from '@automattic/data-stores';
 import './style.scss';
 
+interface HomepagePopoverProps {
+	isAdmin: boolean;
+	template?: Template;
+}
+
+const HomepagePopover = ( { isAdmin, template }: HomepagePopoverProps ) => {
+	const translate = useTranslate();
+	const isEnglishLocale = useIsEnglishLocale();
+	const learnMoreLink = (
+		<ExternalLink
+			href={ localizeUrl( 'https://wordpress.com/support/templates/#template-hierarchy' ) }
+			target="_blank"
+			rel="noopener noreferrer"
+		/>
+	);
+
+	let msg = null;
+	if ( ! isAdmin ) {
+		msg = translate(
+			'Administrators can change the content of this page using the Site Editor. {{learnMoreLink}}Learn more{{/learnMoreLink}}.',
+			{
+				components: {
+					learnMoreLink,
+				},
+			}
+		);
+	} else if ( template ) {
+		msg = translate(
+			'You can change the content of this page by editing the %(templateTitle)s template using the Site Editor. {{learnMoreLink}}Learn more{{/learnMoreLink}}.',
+			{
+				components: {
+					learnMoreLink,
+				},
+				args: {
+					templateTitle: decodeEntities( template.title.rendered || template.slug ),
+				},
+			}
+		);
+	} else if (
+		isEnglishLocale ||
+		hasTranslation(
+			'You can change the content of this page by editing the template using the Site Editor. {{learnMoreLink}}Learn more{{/learnMoreLink}}.'
```

Comment:
I'd suggest removing `by editing the template` to cut the word count and to make it consistent with the non-admin case above, what do you think?

Reply:
Sounds good!